src/Bookings.js

```diff
@@ -1,21 +1,69 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import NewBookingForm from "./NewBookingForm.js";
+import FakeBookings from "./data/fakeBookings.json";
 
-const Bookings = () => {
-  const search = searchVal => {
-    console.info("TO DO!", searchVal);
+function Bookings() {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [bookings, setBookings] = useState([]);
+  const [typedSearchValues, setTypedSearchValues] = useState([]);
+  
+  const search = (searchVal) => {
+    setTypedSearchValues(searchVal);
   };
 
+  function addNewBooking(newBooking){
+    setBookings([...bookings, newBooking])
+  }
+
+  const filteredBookings = bookings.filter((customerName) => {
+    return (
+      customerName.firstName.toLowerCase().includes(typedSearchValues) ||
+      customerName.surname.toLowerCase().includes(typedSearchValues)
+    );
+  });
+
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        console.log(data);
+        if (data.error) {
+          setErrorMessage(data.error);
+        } else {
+          setBookings(data);
+        }
+
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.log(error)
+      });
```

Comment:
Love to see the error handling!
But what happens in the catch block here, do we not want to tell the user something went wrong?

Reply:
I have updated it now to tell the user that "The requested information is not available at the moment"